modules/farm_fd2_school/src/entrypoints/Vue2/App.vue

```diff
@@ -107,6 +118,12 @@
 export default {
   data() {
     return {
+      date: '2019-06-15',
+      cropSelect: 'RADISH',
+      quantitySelect: 1,
+      unitSelect: 'BUNCH',
+      commentBox: '',
+      plantNum: '',
```

Comment:
Using `''` for the initially selected radio button works. But it may be more consistent to use `-1`, which is a common convention for values that are invalid indices (e.g. many searching algorithms will return `-1` when the value being searched for is not found).